etc/systemd/system/Makefile.am

```diff
@@ -31,5 +31,9 @@ $(systemdunit_DATA) $(systemdpreset_DATA):%:%.in
 		-e 's,@sysconfdir\@,$(sysconfdir),g' \
 		$< >'$@'
 
+install-data-hook:
+	$(MKDIR_P) "$(DESTDIR)$(systemdunitdir)"
+	ln -s /dev/null "$(DESTDIR)$(systemdunitdir)/zfs-import.service"
```

Comment:
can't use touch here?
0 length file doesn't work?

Reply:
No, masking works exactly that way in systemd - this is the same as calling `systemctl mask zfs-import.service`.